src/main/java/appeng/core/sync/AppEngPacket.java

```diff
@@ -35,7 +35,7 @@
 public abstract class AppEngPacket
 {
 
-	AppEngPacketHandlerBase.PacketTypes id;
+	private AppEngPacketHandlerBase.PacketTypes id;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused "id" private field. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1068)